serving/docs/lmi_new/user_guides/README.md

```diff
@@ -0,0 +1,21 @@
+# LMI Backend User Guides
+
+LMI provides backend specific user guides that cover the following topics:
```

Comment:
Needs newline

In markdown, lines without a newline separating them are interpreted as being in the same paragraph. So this puts the text line and the bullet points in the same paragraph. Some systems will parse it right regardless, but mkdocs does not

Reply:
good to know - the markdown preview in github does parse it how i intended. I'll do a pass on the other docs we have added and see if this needs to be fixed elsewhere (separate PR)